enqueue/amqp-ext/0.3/etc/packages/enqueue-amqp.yaml

```diff
@@ -0,0 +1,23 @@
+enqueue:
```

Comment:
> Maybe this shows something we could propose as a new feature for flex: if specific package is installed, install more configuration, else, just "suggest" it.

This might be a solution, though I support the point not to adding conditions to recipes. 

The packs could be a solution to my problem. You do not install the bundle and amqp-ext separately but a `enqueue/amqp-pack` package. It installs the bundle and the transport and puts required configuration where needed